src/ros_filter.cpp

```diff
@@ -46,6 +48,26 @@
 
 namespace RobotLocalization
 {
+
+  //! @brief Compute covariance volume
```

Comment:
Maybe in FilterUtilities.cpp or RosFilterUtilities.cpp? The former is meant to be ROS-agnostic, so this could go there. That, or you can put it in FilterBase.cpp?